ocp_resources/csi_storage_capacity.py

```diff
@@ -0,0 +1,77 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class CSIStorageCapacity(NamespacedResource):
+    """
+    CSIStorageCapacity object. API reference:
+    https://kubernetes.io/docs/reference/kubernetes-api/config-and-storage-resources/csi-storage-capacity-v1/#CSIStorageCapacity
+    """
+
+    api_group = NamespacedResource.ApiGroup.STORAGE_K8S_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        capacity=None,
+        maximum_volume_size=None,
+        node_topology=None,
+        storage_class_name=None,
+        teardown=True,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        **kwargs,
+    ):
+        """
+        Args:
+            name (str): Name of the endpoints resource
+            namespace (str): Namespace of endpoints resource
+            client: (DynamicClient): DynamicClient for api calls
+            capacity (str, optional): value reported by the csi driver
+            maximum_volume_size (str, optional): maximum volume size reported by csi driver
+            storage_class_name (str): storage class name
+            node_topology (LabelSelector, optional): defines which node has access to the storage for which capacity
```

Comment:
`JobTemplateSpec` get `LabelSelector` object or dict?

Reply:
You mean node_topology? It is a dict.

Reply:
Then it should be `node_topology (dict, optional)`
please fix where needed

Reply:
I will update.